src/foam/core/stdlib.js

```diff
@@ -781,6 +781,9 @@ foam.LIB({
             this.comparatorNegate(bType.compare(b, a));
       },
       function comparatorNegate(num) {
+        // When negating comparisons, a -0 may be returned and
```

Comment:
I'm not sure this says exactly what we mean. I think the main points are:

- `num` may be `-0`
- `-0 === 0 ---> true`
- We wish to avoid comparators to return `-0`

I feel the middle point is what's missing from the current explanation.